### PR TITLE
[V2] chore: add cpu and memory usage limits for scheduler extender

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by edreed** [comment]
Please change the commit message to conform to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)

/lgtm cancel

**Comment by nearora-msft** [comment]
> Please change the commit message to conform to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
> 
> /lgtm cancel

Done

File: charts/latest-v2/azuredisk-csi-driver/values.yaml [hunk 1]
```diff
@@ -149,6 +149,13 @@ schedulerExtender:
       monitor:
         enabled: false
   servicePort: 8889
+  resources:
+    requests:
+      cpu: 150m
```
Comment (andyzhangx) [inline charts/latest-v2/azuredisk-csi-driver/values.yaml:154]: hi @nearora-msft do you have some monitoring data for the requests and limits? and also is `300m` cpu limits enough?

Comment (nearora-msft) [reply charts/latest-v2/azuredisk-csi-driver/values.yaml:154]: These are the max cpu and memory usage numbers from the latest Aurora test runs:

20K Stateless tests:
Max memory: 40MB
Max cpu: 139m

1K stateful tests:
Max memory: 152MB
Max CPU: 24m

303 Disks/ 303 Vm test:
Max Memory : 152
Max CPU: 24

I'll also add the link to the dashboards in the PR that I have on aks-rp

**Pull request merged**